packages/features/ee/teams/components/TeamInviteListItem.tsx

```diff
@@ -1,5 +1,5 @@
 import classNames from "@calcom/lib/classNames";
-import { getPlaceholderAvatar } from "@calcom/lib/defaultAvatarImage";
+import { WEBAPP_URL } from "@calcom/lib/constants";
```

Comment:
Relying on this constant to get avatar.png instead of using team logo which can be a huge string impacting page size